### PR TITLE
refactor: uses `ansis` for color support

### PR DESCRIPTION
**Pull request by SukkaW**
The PR attempts to reduce the installation size of the `webpackbar`.

- Replace `chalk` w/ `ansis`
  - https://www.npmjs.com/package/ansis
- Replace `ansi-escapes` w/ `sisteransi`
  - https://www.npmjs.com/package/sisteransi

The installation size is reduced by about 80 KiB.

### CODE REVIEW

File: src/profiler/format.ts [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import prettyTime from "pretty-time";
-import chalk from "chalk";
+import ansis from "ansis";
```
Comment (pi0) [inline src/profiler/format.ts:2]: We might use `getColor` from `consola/utils` which is already a dep

Comment (SukkaW) [reply src/profiler/format.ts:2]: Yeah! Lemme test that!

Comment (SukkaW) [reply src/profiler/format.ts]: @pi0 I've migrated most `ansis` w/ `consola/utils`: [`638c725` (#145)](https://github.com/unjs/webpackbar/pull/145/commits/638c725312efed8531b3b22ce4878ca0f6380cb6)

However, `src/util/cli.ts` use the hex color which `consola/utils` doesn't have support.

Comment (SukkaW) [reply src/profiler/format.ts]: @pi0 Will you add hex color support to `consolas/utils`? Otherwise we can't drop `ansis`.

Comment (SukkaW) [reply src/profiler/format.ts]: @pi0 Will it be a breaking change though?

**Review by pi0 — APPROVED** [review]
Thanks ❤️

**Pull request merged**